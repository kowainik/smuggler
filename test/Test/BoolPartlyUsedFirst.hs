module Test.BoolPartlyUsedFirst where

import Data.Bool (bool, not)

foo = bool
