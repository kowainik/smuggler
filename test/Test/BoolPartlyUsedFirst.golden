module Test.BoolPartlyUsedFirst where

import Data.Bool (bool)

foo = bool
